examples/msbr/openmc_msbr_model.py

```diff
@@ -0,0 +1,1045 @@
+import openmc
+import numpy as np
+import argparse
+
+from openmc.deplete import CoupledOperator, PredictorIntegrator, CELIIntegrator
+
+from _core_elements import *
+from _control_rods import *
+from _root_geometry import *
+
+# Materials
+
+fuel = openmc.Material(name='fuel')
+fuel.set_density('g/cm3', density=3.35)
+fuel.add_components({'Li7': 0.0787474673879085,
+                     'Be9': 0.0225566879138321,
+                     'F19': 0.454003012179284,
+                     'Th232': 0.435579130482336,
+                     'U233': 0.00911370203663893},
```

Comment:
Magic numbers? Seems dangerous to have hardcoded values with such high precision.

Reply:
These values are based off the nuclide wt-% in the Serpent model.

Reply:
Can you read it in from a json file or something? It's hard to look at these numbers and say "hey that's reasonable" without knowing where they come from. I don't even know where to look to check that they're correct.

Reply:
The associated data is in `examples/msbr/mats/msbr_saltproc_prepr_comp.ini` where the values are also hardcoded but have a different "key."

I still want to know why these numbers and where they came from. The file above does not have this information either.

Reply:
That information is contained in my thesis. If you check the readme you'll see that there is a reference to it.

Reply:
The readme for your thesis or for saltproc? I don't see any references to your (unpublished?) thesis in the saltproc readme. Perhaps it's in another PR?

Reply:
It should be at `[examples/msbr/README.md](https://github.com/arfc/saltproc/pull/165/files#diff-7269b075db39601e1335dff3623870154ac78461528b87a7ee0d3b355a9fb38d)`.

Reply:
Chapter 4 isn't done yet, but the part containing the model description is. If you want to read it you can do so here:
[main.pdf](https://github.com/arfc/saltproc/files/10158521/main.pdf)